src/assets/styles/layout.scss

```diff
@@ -53,7 +53,7 @@
 }
 
 .section-container {
-  padding: 1rem 1rem;
+  padding: 1.25rem 1.25rem;
```

Comment:
Requirements call for more padding on these subsections. 
Verified in Alterations, Change and Correction filings